kustomize/commands/internal/util/util_test.go

```diff
@@ -99,6 +99,15 @@ func TestGlobPatternsWithLoaderRemoteFile(t *testing.T) {
 	if len(resources) > 0 {
 		t.Fatalf("incorrect resources: %v", resources)
 	}
+
+	// test load unreachable remote file with skipped verification
+	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{invalidURL}, true)
+	if err != nil {
+		t.Fatalf("unexpected load error: %v", err)
+	}
+	if len(resources) != 1 || resources[0] != invalidURL {
+		t.Fatalf("incorrect resources: %v", resources)
+	}
```

Comment:
It might make sense for this test (and the previous ones) to use the [`require` primitives from Testify](https://pkg.go.dev/github.com/stretchr/testify/require#NoError). For example (illustrative only, as it'd require more changes):

```suggestion
	resources, err = GlobPatternsWithLoader(fSys, ldr, []string{invalidURL}, true)
	require.NoError(t, err, "unexpected load error")
	require.Equal(t, 1, len(resources))
	require.NotEqual(t, invalidURL, resources[0], "incorrect resources")
```

`require` already fails fast, thus removing the need for conditions.